docs/how-to-use-kit.md

```diff
@@ -0,0 +1,105 @@
+* Based on these samples, a users can create their own pipelines for specific tasks based on their test use case and run these pipelines using pipeline runs.
+
+
+How to collect metrics
+
+* Access the Grafana dashboard on [http://localhost:8080](http://localhost:8080/), make sure you have run port-forward command mentioned above.
+* All the core Kubernetes components like API Server, KCM, scheduler, etcd, authenticator are being scraped and their metrics are being plotted in their own Grafana dashboards
+* For node level metrics in Grafana refer to NodeExporter/Nodes dashboard
+
+Adding additional dashboards from Github repo-
+
+```bash
+# wget grafana dashboard from the repo
+wget https://raw.githubusercontent.com/awslabs/kubernetes-iteration-toolkit/main/substrate/monitoring/master-dashboard.json
+
+# Create configmap in management cluster to load this dashboard
+kubectl create configmap master-dashboard -n monitoring --from-file=master-dashboard.json
+kubectl label configmap master-dashboard -n monitoring  grafana_dashboard=1
+```
+>Note: we are [tracking the task](https://github.com/awslabs/kubernetes-iteration-toolkit/issues/207) on automating this step as well where the Grafana dashboards are pre-loaded in Grafana
+
+Accessing logs for kube components-
+[WIP](https://github.com/awslabs/kubernetes-iteration-toolkit/issues/106)
+
+### Use-cases supported with KIT
+
+* As a Kubernetes developer/user, create x number of objects(pods, secrets, config maps) in a cluster. As part of the validation, I want to
+    * Check API server, scheduler throughput and etcd usage, make sure resource utilization(s) are below threshold
+* As a Kubernetes developer/user, I want to tweak a flag in Kubernetes control plane and deploy some workload. As part of the validation, I want to
+    * Check that the latency for the API server calls is not impacted/SLO’s are not impacted.
+    * Check API server and etcd usage, make sure things are below threshold
+* As a Kubernetes developer/user, In an EKS cluster, I want to run an e2e scalability test to create x number of different objects and y number of nodes 
+    * Check the SLO and latency
+* As a Kubernetes developer/user, I have a custom API server/Scheduler image with some changes and want to test these change in a Kubernetes cluster, by creating x number of objects
+* As a Kubernetes developer/user, I want to run etcd or master components on specific EC2 instance types and run scale tests
+* As a Kubernetes developer/user, in an existing EKS cluster can I run a load test, get all the metrics, resource usage and validation tests
+
+
+## Key Terms
+
+**KIT/clusters or guest clusters -** These are rapid prototyping vanilla Kubernetes clusters provisioned using using EKS-distro images. They take about 3-4 minutes to provision on ec2 nodes and less than 30 seconds to update their configurations. These clusters are provisioned by KIT-operator running in the KIT environment
```

Comment:
Nit: should this be "KIT clusters or guest clusters"?

Reply:
We have been using these interchangeably, so want to make sure both are covered.

Reply:
Line 41 references quip which is AWS internal only ;)